### ISSUE TITLE
Persistent Working Directory

### ISSUE THREAD

**Issue by fkorotkov** [cirruslabs/cirrus-ci-docs#85]
It will be nice to have an option to persist working directory between tasks. That way it will be possible to populate a working directory in one tasks and use it in parallel sub-sequential tasks.

**Comment by cevich** [comment]
With parallel tasks, this is an incredibly challenging feature to pull off in a way that is predictable, data-consistent and generally applicable.  Not to mention using/supporting multiple clouds and technologies (VMs vs Containers).  Since Cirrus is already capable of some basic synchronization (task-dependencies).  In many/most cases, dependencies, encrypted values, and existing cloud storage services should suffice.  So IMHO, this feature can probably be closed as "not worth the effort".

**Comment by fkorotkov** [comment]
@cevich agreed! This feature won't be available though out all the supported clouds. My initial thought was just to support it for Kubernetes tasks via Persistent Volumes and Volume Snapshots. It's super challenging and definitely not happening in the near future 😅

**Comment by fkorotkov** [comment]
This will be taken care of by #316
